src/main/webroot/upload/csv/index.html

```diff
@@ -40,14 +40,22 @@ <h1 id="csv-upload-title">CSV Upload</h1>
           <form class="form-inlin justify-content-center" action="/collections" method="post"
             enctype="multipart/form-data">
             <div class="form-group">
-              <label for="csv-file">File to upload:</label> <input type="file" name="csv-file" id="csv-file">
+              <label for="csv-file">File to upload:</label> <input type="file" name="csv-file" id="csv-file" required>
             </div>
             <div class="form-group">
               <label for="iiif-host">IIIF Host:</label> <input type="text" value="https://iiif.library.ucla.edu/iiif/2"
-                name="iiif-host" id="iiif-host" onfocus="this.value=''" />
+                name="iiif-host" id="iiif-host" onfocus="this.value=''" required/>
               <div class="infotext">Use the default IIIF URL (i.e., https://iiif.library.ucla.edu/iiif/2) or enter the
                 base URL for a different IIIF server.</div>
             </div>
+            <div class="form-group">
+              <label for="iiif-version">IIIF manifest version:</label> 
+	      <select name="iiif-version" id="iiif-version" required>
```

Comment:
Does the `required` do anything here? One of those three options is going to be submitted regardless.

I'm not sure what will happen when we load the OpenAPI spec and a blank value is submitted. The router may return an error since that value wouldn't conform to the spec? It will be something to test, I guess, once we hook that up.

It might also be interesting to get some user feedback (perhaps from Dawn and Lisa) about their preferences for a default value of 'v2' vs. the empty default value, if you haven't already. I'm not saying that needs to be done now... just that we might want to inquire before the publication of it (again, only if you didn't already). And if they did prefer a non-empty default value, at what point would we change that?

Anyway, looks good for now.